CONTRIBUTING.md

```diff
@@ -7,75 +7,66 @@ The following are a set of guidelines for contributing to the project.
 #### Contents
 
 * [Code of Conduct](#code-of-conduct)
-* [Ways of Contributing](#ways-of-contributing)
-  * [Joining Feature Discussions](#joining-feature-discussions-) 💭
-  * [Joining Developer Discussions](#joining-developer-discussions-) 💬
-  * [Reporting Bugs](#reporting-bugs-) 🐛
-  * [Requesting Features](#requesting-features-) 🚀
+* [Ways to Contribute](#ways-to-contribute)
+  * [Questions or Enhancement Requests](#questions-or-enhancement-requests-) 💭
+  * [Reporting and Fixing Bugs](#reporting-and-fixing-bugs-) 🐛
+  * [Submitting Features](#submitting-features-) 🚀
   * [Contributing Code](#contributing-code-) 👾
```

Comment:
The contributing code section seems like it could now be moved outside of "Ways to Contribute" and just referenced from "Reporting and Fixing Bugs" or "Submitting Features" as the practical guide for creating a pull request etc.